README.md

```diff
@@ -19,7 +19,7 @@ dev #default branch
 
 This website is compiled using node, using Docusaurus framework components
 
-1. Download and install nodejs (version>12.5.0)
+1. Download and install nodejs (Node >=16.15.1 <= 18)
```

Comment:
I remember node seems to only require node16.14+, I'm not sure if 16.15 is required

Reply:
no Node, I think version/node.version is better

Reply:
> no Node, I think version/node.version is better没有节点，我认为version/node.version更好

ditto，just like https://github.com/sass/node-sass/releases/tag/v4.14.1

Reply:
Hi! @cjj2010 Thanks for your reminder. docusaurus uses version 2.4.3. After we tested, it can run normally only if the nodejs version is greater than 16.14.There will be a warning when you run the project below v16.14.Should we change it to >=16.14.x <=18?